### PR TITLE
falsey bug

### PR DESCRIPTION
**Pull request by anntisa**
half of the changes are due to Android Studio updating.
I will comment on the places that were actually changed.

### CODE REVIEW

File: survata-sample/src/main/java/com/survata/demo/ui/DemoFragment.java [hunk 1]
```diff
@@ -67,10 +68,7 @@ public void checkSurvey() {
 
         final Context context = getContext();
         String publisherId = Settings.getPublisherId(context);
-        SurveyDebugOption option = new SurveyDebugOption(publisherId);
-        option.preview = Settings.getPreviewId(context);
-        option.zipcode = Settings.getZipCode(context);
-        option.sendZipcode = Settings.getZipCodeEnable(context);
+        SurveyOption option = new SurveyOption(publisherId);
```
Comment (anntisa) [inline survata-sample/src/main/java/com/survata/demo/ui/DemoFragment.java]: fix survata sample to not use survey preview

File: survata-sdk/build.gradle [hunk 2]
```diff
@@ -41,7 +41,7 @@ publish {
     userOrg = 'survata'
     groupId = 'com.survata.android'
     artifactId = 'library'
-    publishVersion = '1.0.13'
+    publishVersion = '1.0.14'
```
Comment (anntisa) [inline survata-sdk/build.gradle]: new version!

File: survata-sdk/src/main/java/com/survata/SurveyOption.java [hunk 3]
```diff
@@ -37,7 +37,9 @@ public SurveyOption(String publisher) {
         map.put("explainer", explainer);
         map.put("contentName", contentName);
         map.put("preview", preview);
-        map.put("testing", Boolean.toString(testing));
+        if (testing) {
+            map.put("testing", Boolean.toString(testing));
+        }
```
Comment (anntisa) [inline survata-sdk/src/main/java/com/survata/SurveyOption.java]: falsey bug

File: survata-sample/src/main/java/com/survata/demo/ui/DemoFragment.java (hunk 1 above)
Comment (jameswwarner) [inline survata-sample/src/main/java/com/survata/demo/ui/DemoFragment.java]: We might want to leave survata sample as is.

File: survata-sdk/src/main/java/com/survata/SurveyOption.java (hunk 3 above)
Comment (jameswwarner) [inline survata-sdk/src/main/java/com/survata/SurveyOption.java]: :+1:

File: survata-sample/src/main/java/com/survata/demo/ui/DemoFragment.java (hunk 1 above)
Comment (anntisa) [inline survata-sample/src/main/java/com/survata/demo/ui/DemoFragment.java]: ok i have changed it back. i added the SurveyOption line as a comment instead

**Comment by jameswwarner** [comment]
LGTM. 🚢

**Pull request merged**